### ISSUE TITLE
MobX support

### ISSUE THREAD

**Issue by sserdyuk** [staydecent/nativescript-preact#3]
If we're thinking about how this would work, settling on MobX for state store would be super cool. The original 2-way binding in NS leads to fragmenting state into many pieces and also has holes of inconsistencies in its template expression rendering, and while it's possible to make it all work, I felt like this days we need a cleaner "react like" approach.

**Comment by staydecent** [comment]
Agreed. The goal is to hide the NS Observables and binding and allow Preact `setState`, Redux, MobX etc. to work out of the box.

Once the Preact/Undom to NS bindings are handled by this library there should be no extra work to get Redux nor MobX working out of the box.
